source/js/components/details.js

```diff
@@ -46,6 +46,8 @@ var Details = React.createClass({
 
     if (key) {
       hash = `#` + key;
+    } else {
+      hash = `#details`;
```

Comment:
Ah, good idea. This makes me think of one step further?

`document.querySelector(`.menu-link.active`).getAttribute(`href`)`

Might be a bit better to get the active link (more likely to be the right one) than the last one. The last one assumes the accordion is always near the bottom of the page, which maybe we change it's place in the nav?